Directory.Build.props

```diff
@@ -71,20 +75,11 @@
     -->
     <When Condition="'$(DotNetBuildSourceOnly)' == 'true' AND '$(DotNetBuildOrchestrator)' == 'true'">
       <PropertyGroup>
-        <DefaultNetCoreTargetFramework>$(NetCurrent)</DefaultNetCoreTargetFramework>
-        <DefaultNetCoreTargetFrameworks>$(DefaultNetCoreTargetFramework)</DefaultNetCoreTargetFrameworks>
+        <NetVS>$(NetCurrent)</NetVS>
+        <NetVSCode>$(NetCurrent)</NetVSCode>
+        <NetVSAndVSCode>$(NetVS);$(NetVSCode)</NetVSAndVSCode>
```

Comment:
This looks wrong. When building the product from source, only the latest TFM should get built. See roslyn's defines as an example: https://github.com/dotnet/roslyn/blob/c02ed62551e0880585f14b511cc1b9ba4c48bcf2/eng/targets/TargetFrameworks.props#L53-L61

Is there any reason why those older TFMs got added? cc @dotnet/source-build @jaredpar

Reply:
So only `$(NetCurrent)` should be built? I can fix

Reply:
Yes